examples/http/server.cc

```diff
@@ -31,17 +35,16 @@ class RequestHandler : public HTTP_SERVER_NS::HttpRequestCallback
     options.parent   = opentelemetry::trace::propagation::GetSpan(new_context)->GetContext();
 
     // start span with parent context extracted from http header
-    auto span =
-        get_tracer("http-server")
-            ->StartSpan(
-                span_name,
-                {{"http.server_name", server_name},
-                 {"net.host.port", server_port},
-                 {"http.method", request.method},
-                 {"http.scheme", "http"},
-                 {"http.request_content_length", static_cast<uint64_t>(request.content.length())},
-                 {"http.client_ip", request.client}},
-                options);
+    auto span = get_tracer("http-server")
+                    ->StartSpan(span_name,
+                                {{SemanticConventions::GetAttributeHttpServerName(), server_name},
```

Comment:
It looks rather bulky, in my opinion.

Reply:
Agree. Do we have a better way of handling this?

Reply:
There are several options:
- make methods `constexpr` and return a string literal constant, inited as const char * or char array
- or just define all string constants in a dedicated namespace, that enables `using` for those who want to write it in a shorter form. Then for gcc / clang this scenario is already optimized, and for msvc we need to enable string pooling to combine all strings into one region of memory. These constants all have global static duration, there is no need to "get" them, they'd be efficiently compile-time optimized. Using `string_view` for that is an overkill.

Reply:
Thanks for the suggestion. I did initially wanted to use the second option but wasn't aware of the compiler optimization of this static global across multiple translation units. Let me try that.

Reply:
As mentioned in prev comment, have created the `constexpr` function now.